Cargo.toml

```diff
@@ -38,9 +39,11 @@ opentelemetry-prometheus = "0.16.0"
 prometheus = "0.13.4"
 serde = { version = "1.0.202", features = ["derive"] }
 serde_json = "1.0.117"
-masking = { git = "https://github.com/juspay/hyperswitch", tag = "v1.109.0" }
+masking = { git = "https://github.com/juspay/hyperswitch.git",branch = "update_scylla",features = ["cassandra"] }
```

Comment:
Can we avoid the branch reference?

Reply:
This is temporary will remove once the PR is merged

Reply:
Done removed